src/components/DeploymentIconText.vue

```diff
@@ -33,4 +33,12 @@
   const deploymentId = computed(() => props.deploymentId)
   const { deployment, subscription } = useDeployment(deploymentId)
   const deploymentName = computed(() => deployment.value?.name)
-</script>
\ No newline at end of file
+</script>
+
+<style>
+.deployment-icon-name { @apply
+  truncate
+  max-w-48
+  block
+}
+</style>
```

Comment:
Ah wait actually - i don't know this is correct - we'd want to put this sort of styling on the parent of this component and not the component itself, setting only the styles this component actually needs. So in this case it's probably `max-w-full`

Reply:
I'm not sure I follow you exactly.  What I have added is working though.  

Updated:

<img width="1133" alt="image" src="https://github.com/user-attachments/assets/117da7c8-91ba-4a41-9850-493a29913f8b" />


Previous:

<img width="1125" alt="image" src="https://github.com/user-attachments/assets/2990d471-880e-4dc9-b646-d861401198b6" />